pkg/cmd/printer/broadcast.go

```diff
@@ -0,0 +1,56 @@
+package printer
+
+import (
+	"context"
+
+	"github.com/aquasecurity/tracee/pkg/logger"
+	"github.com/aquasecurity/tracee/types/trace"
+)
+
+// Broadcast is a printer that broadcasts events to multiple printers
+type Broadcast struct {
+	eventsChan []chan trace.Event
+}
+
+// NewBroadcast creates a new Broadcast printer
+func NewBroadcast(ctx context.Context, printers []EventPrinter) *Broadcast {
+	eventsChan := make([]chan trace.Event, 0, len(printers))
+
+	for _, printer := range printers {
+		// we use a buffered channel to avoid blocking the event channel, but with a small buffer
+		eventChan := make(chan trace.Event, 5)
+		eventsChan = append(eventsChan, eventChan)
+
+		go startPrinter(ctx, eventChan, printer)
+	}
+
+	return &Broadcast{eventsChan: eventsChan}
+}
+
+// Print broadcasts the event to all printers
+func (b *Broadcast) Print(event trace.Event) {
+	for _, c := range b.eventsChan {
+		// we use select to avoid blocking the print loop if one of the printers is slow
+		// in case the event can't be sent because the buffered channel is full, we drop it
+		// and log an error
+		select {
+		case c <- event:
+		default:
+			logger.Error("dropping event due to slow printer", "event", event)
+		}
```

Comment:
I got more error logs than events, running:

sudo ./dist/tracee -t e='*' --output table:stdout

This logger Error polutes table output, WDYT of changing it to debug level and to increment errors channel?

Reply:
@geyslan Done. Though if you test with "e='*'" we will lose events just like we do on the main branch, and possibly the  files generated with two prints will have different event on it.